### ISSUE TITLE
Add button to the city screen to toggle the display of present and/or produced yields

### ISSUE THREAD

**Issue by devolution79** [We-the-People-civ4col-mod/Mod#813]
![image](https://user-images.githubusercontent.com/32330976/231443696-0ef1cbe3-7344-46ae-a1b1-a74dea4f0dd6.png)

**Comment by devolution79** [comment]
TODO: Need translation for the hoover text. Currently it is hard-coded as: "Show or hide yields"

**Comment by MrZorG33** [comment]
By the way, is it possible to change this button to be like this?
![image](https://user-images.githubusercontent.com/61798883/231446634-656ee0d0-918f-4378-b258-cb03d5880daa.png)

**edit**: the yield panel can be moved a little to the right and there will be room for a large button.

**Comment by devolution79** [comment]
Certainly :) The current positioning / layout is just a placeholder.

**Comment by raystuttgart** [comment]
@devolution79 
The German translation could be:
 **"Waren ein-/ausblenden"**

**Comment by MrZorG33** [comment]
@devolution79 oh, I just thought: do you need a translation so that you write it down? or will we have a txt_key tag?

**Comment by raystuttgart** [comment]
@devolution79 
What is the status of this?
Is it already integrated in "develop"?

**Comment by devolution79** [comment]
Currently blocked by the ugliness of the button placement :P Perhaps we should go with a rectangle aligned with the first yield and second yield row ?

**Comment by devolution79** [comment]
Maybe move it to the black circle location now to get unblocked: 
![image](https://user-images.githubusercontent.com/32330976/235880973-495223ae-bde5-47d8-8e58-9e8beebb643b.png)

**Comment by MrZorG33** [comment]
By the way, the players are asking if it is possible to do not completely hide yields, but make them barely noticeable (transparent)?

**Comment by devolution79** [comment]
Moved the button and added English/German support. Unless there are any objections we could merge ?
Note: Decided to make it a toggle button and to use the glow overlay (like the avoid growth button)  when activated

**Comment by Nightinggale** [comment]
Merged into develop 148cd68deb08de6facedd0de8466c79982025efd
